CarpetLib/src/interpolate_3d_4tl.cc

```diff
@@ -115,11 +115,10 @@ void interpolate_3d_4tl(T const *restrict const src1, CCTK_REAL const t1,
   RT const s4fac =
       (t - t1) * (t - t2) * (t - t3) / ((t4 - t1) * (t4 - t2) * (t4 - t3));
 
-// Loop over region
-#pragma omp parallel
+  // Loop over region
+  // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.